model/types.go

```diff
@@ -191,15 +191,32 @@ type Volume struct {
 	FcSessions     []*FcSession           `json:"fc_sessions,omitempty"`
 }
 
+// Workaround NOS 5.0.x vs 5.1.x responses with different case
 // FcSession info
 type FcSession struct {
-	InitiatorWwpn string `json:"initiator_wwpn,omitempty"`
+	InitiatorWwpn       string `json:"initiator_wwpn,omitempty"`
+	InitiatorWwpnLegacy string `json:"initiatorWwpn,omitempty"`
 }
 
 // IscsiSession info
 type IscsiSession struct {
-	InitiatorName string `json:"initiator_name,omitempty"`
-	InitiatorIP   string `json:"initiator_ip_addr,omitempty"`
+	InitiatorName       string `json:"initiator_name,omitempty"`
+	InitiatorNameLegacy string `json:"initiatorName,omitempty"`
+	InitiatorIP         string `json:"initiator_ip_addr,omitempty"`
+}
+
+func (s FcSession) InitiatorWwpnStr() string {
```

Comment:
Can you drop the Str?

Reply:
@rgcostea attribute name and function names will clash. Hence added Str. Let me know if we can call it something else?

Reply:
wow.  i miss overloading.  I guess this name will have to do :)

Reply:
:-)